File: 코딩 기초 트레이닝/배열에서 문자열 대소문자 변환하기-p181875/배열에서 문자열 대소문자 변환하기.py
# https://school.programmers.co.kr/learn/courses/30/lessons/181875

def solution(strArr):
    # 홀수 인덱스 대문자
    # 짝수 인덱스 소문자

    answer = []

    for index, s in enumerate(strArr):
        if index % 2 == 0 :
            answer.append(s.lower())
        else:
            answer.append(s.upper())

    return answer

File: 코딩 기초 트레이닝/배열에서 문자열 대소문자 변환하기-p181875/배열에서 문자열 대소문자 변환하기.js
// https://school.programmers.co.kr/learn/courses/30/lessons/181875

function solution(strArr) {
    var answer = [];

    strArr.forEach((s, index) => {
        if(index % 2 === 0) {
            answer.push(s.toLowerCase());
        } else {
            answer.push(s.toUpperCase());
        }
    });

    return answer;
}
